lib/file.js

```diff
@@ -205,7 +205,13 @@ internals.openStat = function (path, mode, callback) {
     Fs.open(path, mode, function(err, fd) {
 
         if (err) {
-            return callback(Boom.notFound());
+            if (err.code === 'ENOENT') {
+                return callback(Boom.notFound());
+            } else if (err.code === 'EACCES' | err.code === 'EPERM') {
```

Comment:
I used a single `|` to retain code coverage. The new tests will trigger `EACCES` on linux but on windows `EPERM` is triggered instead. Afaik both values indicate a permission issue.